addons/sourcemod/scripting/surftimer/commands.sp

```diff
@@ -2251,6 +2265,17 @@ void PrespeedText(int client, bool menu = false)
 		MiscellaneousOptions(client);
 }
 
+void HintsText(int client, bool menu = false)
+{
+	if (g_bAllowHints[client])
```

Comment:
`g_bAllowHints[client] = !g_bAllowHints[client];`
in a smaller easier version instead of the if/else

Reply:
I orginally just copied the prestrafe functions. Changed that one as well